org.eclipse.winery.frontends/app/tosca-management/src/app/instance/sharedComponents/validSourceTypes/validSourceTypes.component.html

```diff
@@ -16,17 +16,17 @@
 </div>
 <div *ngIf="!loading">
     <div class="right">
-        <button class="btn btn-primary" name="save" (click)="saveToServer()"
+        <button class="btn btn-primary" name="save" style="margin-bottom: 10px;" (click)="saveToServer()"
                 [disabled]="!sharedData?.currentVersion?.editable">
             Save
         </button>
     </div>
 
-    <winery-table [title]="title"
-                  [columns]="columns"
-                  [data]="validSourceTypes?.nodes"
-                  (addBtnClicked)="onAddClick()"
-                  (removeBtnClicked)="onRemoveClicked($event)">
+    <winery-table
+        [columns]="columns"
```

Comment:
Move this to `<winery-table...` level and reformat again pls.

Reply:
Should be fixed now